### DISCUSSION TITLE
Hidden Configure columns on Table

### DISCUSSION THREAD

**Discussion by hoangnamitc** [orchidsoftware/platform#2340] [Q&A] [answered]
<img width="319" alt="image" src="https://user-images.githubusercontent.com/7042462/185748714-1d44809a-4996-4274-a412-1d93c3d4cc83.png">

How to hidden this?

Thanks!

**Comment by c-gross** [comment, marked as answer]
It should be gone if you add to every column of the table `cantHide()`.

**Reply by hoangnamitc** [reply]
This is working. Thanks !
